internal/api/block_documents.go

```diff
@@ -29,16 +29,13 @@ type BlockDocument struct {
 	BlockType     BlockType `json:"block_type"`
 
 	BlockDocumentReferences string `json:"block_document_references"`
-
-	IsAnonymous bool `json:"is_anonymous"`
 }
 
 type BlockDocumentCreate struct {
-	Name          *string   `json:"name"` // names are optional for anonymous blocks
+	Name          string    `json:"name"`
```

Comment:
per our convo and confirming w the backend team, anonymous blocks are a server-side implementation and don't exist functionally for outside users